Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'bootsnap', '>= 1.4.2', require: false
+gem 'rest-client'
+gem 'webmock'
```

Comment:
This gem is NOT going to be used in production. Please move to `:development, :test`

Reply:
This is now moved accordingly!